CloneTiving/CloneTiving/ViewControllers/Setting/SettingViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  SettingViewController.swift
+//  CloneTiving
+//
+//  Created by 지희의 MAC on 2023/05/01.
+//
+
+import UIKit
+import SnapKit
+
+class SettingViewController: BaseViewController {
+    
+    private let settingView = SettingTableView()
+    private let settingHeaderView = SettingHeaderView()
+    private let logOutButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("로그아웃", for: .normal)
+        button.setTitleColor(.tvingGray2, for: .normal)
+        button.titleLabel?.font = .tvingRegular(ofSize: 14)
+        button.makeBorder(width: 1, color: .tvingGray4)
+        button.makeCornerRound(radius: 5)
+        return button
+    }()
+    
+    
+    let cellIdentifier = "settingCell"
+    let settingTexts: [String] = ["이용권", "1:1 문의내역", "예약알림", "회원정보 수정", "프로모션 정보 수신 동의"]
+    let settingTexts2: [String] = ["공지사항", "이벤트", "고객센터", "티빙 알아보기"]
```

Comment:
```suggestion
    let settingTexts = [["이용권", "1:1 문의내역", "예약알림", "회원정보 수정", "프로모션 정보 수신 동의"],["공지사항", "이벤트", "고객센터", "티빙 알아보기"]]
```
이부분을 `section`을 이용하기 위해서 하나의 list로 묶어보는건 어떨까요 자세한 이유는 아래에 `tableViewDataSource`에 남겨놓겠습니다